src/openfermion/utils/__init__.py

```diff
@@ -18,6 +18,8 @@
 
 from ._hubbard import fermi_hubbard
 
+from ._meanfield import meanfield_dwave
```

Comment:
I missed this last time: imports should be alphabetical (i.e. jellium imports should come before meanfield). But I won't let this hold things up.